imports/api/meters/meters.js

```diff
@@ -16,14 +16,14 @@ import { ActivePeriod } from '/imports/api/behaviours/active-period.js';
 export const Meters = new Mongo.Collection('meters');
 
 Meters.readingSchema = new SimpleSchema({
-  date: { type: Date, autoform: { type: 'datetime-local', defaultValue() { return Clock.currentTime(); } } },
+  date: { type: Date, autoform: { type: 'datetime-local', defaultValue() { return Clock.currentDate(); } } },
```

Comment:
Azt hiszem ha atallunk sima Date-re datetime helyett , akkor a datetime-local-rol is at kell allnunk sima date-re, es akkor az UTC-ben fog lemenni, nem lesz gond a +/- 2 ora miatt nem tud datumot ugrani